webpack/components/extensions/HostDetails/ActionsBar/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { DropdownGroup, DropdownItem } from '@patternfly/react-core';
+import { CubeIcon } from '@patternfly/react-icons';
+
+import { translate as __ } from 'foremanReact/common/I18n';
+import { foremanUrl } from 'foremanReact/common/helpers';
+
+import { selectHostDetails } from '../HostDetailsSelectors';
+
+const HostActionsBar = () => {
+  const hostDetails = useSelector(selectHostDetails);
+
+  return (
+    <DropdownGroup label={__('Katello')} key="katello-group">
```

Comment:
This will end up as 'Satellite' downstream which doesn't make much sense. I'd suggest either eliminating the group, or maybe 'Content' is a better heading.
```suggestion
    <DropdownGroup label={__('Content')} key="katello-group">
```